### ISSUE TITLE
Terminal input does not work with NVDA

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#121735]
Version: 1.56.0-insider (system setup)
Commit: b4816cfd1f4161a3bb272354d181a9947760ee26
Date: 2021-04-20T07:23:51.425Z
Electron: 12.0.4
Chrome: 89.0.4389.114
Node.js: 14.16.0
V8: 8.9.255.24-electron.0
OS: Windows_NT x64 10.0.19042

Steps to Reproduce:

1. Load NVDA.
2. Press ctrl+` to create a new terminal.
3. In the terminal input area, type `exit` (enter).

CMD or PowerShell terminal does not disappear. Any other command is not passed to the terminal.

Does this issue occur when all extensions are disabled?: Yes/No

Yes

**Comment by jooyoungseo** [comment]
CC @isidorn.

**Comment by isidorn** [comment]
@jooyoungseo is this a recent breakage? Since I see you are using insiders?

Assigning to @Tyriar and @meganrogge since they have been doing terminal changes recently

**Comment by jooyoungseo** [comment]
@isidorn, yes, this is a recent bug.

**Comment by isidorn** [comment]
@Tyriar @meganrogge maybe the focus movement changed with the Terminal Tab area.
Also keep in mind accessibility for the terminal tab area. If it helps I can do a test pass...

**Comment by Tyriar** [comment]
This looks like the electron winpty breakage.
